### PR TITLE
Dockerize project

### PR DESCRIPTION
**Pull request by jasondavindev**
- Add docker compose file to dockerize application and database
- Add shell script to create database user with required accesses (as no root user)
- Add shell scripts to help devs with UNIX based OS (macOS and Linux)
- Update README

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  db:
+    image: mongo:4.2
+    volumes:
+      - $PWD/docker/volumes/db:/data/db
+      - $PWD/docker/scripts:/docker-entrypoint-initdb.d
+    networks:
+      - development_network
+    env_file: .env
+
+  app:
```
Comment (powilliam) [inline docker-compose.yml:13]: I'm getting the following error. Can you handle it?

```ERROR: for twitch_app_1  Cannot create container for service app: create .: volume name is too short, names should be at least two alphanumeric characters

ERROR: for app  Cannot create container for service app: create .: volume name is too short, names should be at least two alphanumeric characters
ERROR: Encountered errors while bringing up the project.```

**Review by powilliam — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (jasondavindev) [reply docker-compose.yml:13]: What's your docker and docker compose version?

Comment (powilliam) [reply docker-compose.yml:13]:
``` bash
[powilliam@dev ~]$ docker -v
Docker version 19.03.8-ce, build afacb8b7f0
[powilliam@dev ~]$ docker-compose -v
docker-compose version 1.25.5, build unknown
```

Comment (jasondavindev) [reply docker-compose.yml:13]: calling `echo $(pwd)` in your current directory, what's ouput?

Comment (powilliam) [reply docker-compose.yml:13]: It's the relative path to the folder. The problem is that docker and docker-compose isn't configured correctly. I fixed the problem. Sorry about that, i'll approve right know.

**Review by powilliam — APPROVED** [review]

**Pull request merged**